src/base/_themes.scss

```
@@ -37,6 +38,7 @@
   --theme-color-text-base: #{color.$text-light};
   --theme-color-text-code: var(--theme-color-text-emphasis);
   --theme-color-text-emphasis: #{color.$text-light-emphasis};
+  --theme-color-text-muted: #{color.$base-white};
```

```suggestion
  --theme-color-text-muted: inherit;
```

As-is, this will probably unintentionally make the text slightly brighter than `$text-light`. If the intention is for it to retain the parent text color, maybe we can just make `inherit` the default?